app/assets/stylesheets/forever_style_guide/globals/_effects.scss

```diff
@@ -161,3 +161,37 @@ $color-card-bg: color('white');
     }
   }
 }
+
+// can be added to sections to create slant effect
+@each $id, $color in $colors {
+
+  .slant-right-#{$id} {
+  width: 100%;
+  position: relative;
+
+    &::after {
+      content: "";
+      position: absolute;
+      width: 0;
+      height: 0;
+      border-right: 100vw solid transparent;
+      border-bottom: 60px solid color('#{$id}');
+      top: -60px;
+    }
+  }
+
+  .slant-left-#{$id} {
+  width: 100%;
+  position: relative;
+  
+    &::after {
+      content: "";
+      position: absolute;
+      width: 0;
+      height: 0;
+      border-left: 100vw solid transparent;
+      border-bottom: 60px solid color('#{$id}');
```

Comment:
I would suggest replacing all of thes 60's w/ a variable or, probably better, making this into a mixin that accepts a value to define how steep you want the effect. Alternatively, you could provide a standard rhythm for decreasing the steepness at given breakpoints.

In the current implementation it seems way too steep at any non-desktop breakpoint.

<img width="1415" alt="screen shot 2017-08-07 at 5 36 18 pm" src="https://user-images.githubusercontent.com/3814998/29046834-34334476-7b97-11e7-81c9-28064c900d76.png">